grails-app/i18n/messages.properties

```diff
@@ -1547,7 +1547,7 @@ report.consumedQuantity.label=Cnsm
 report.containerNumber.label=Container #
 report.damagedQuantity.label=Dmg
 report.dateRange.label=Date range
-report.dayReplenishement.label=day replenishement
+report.dayReplenishement.label=day replenishment
```

Comment:
I guess we could change the "report.dayReplenishement.label" to "report.dayReplenishment.label" too :)